data.sql

```diff
@@ -35,3 +35,149 @@ UPDATE animals SET owner_id = 4 WHERE name LIKE 'Squirtle';
 UPDATE animals SET owner_id = 4 WHERE name LIKE 'Blossom';
 UPDATE animals SET owner_id = 5 WHERE name LIKE 'Angemon';
 UPDATE animals SET owner_id = 5 WHERE name LIKE 'Boarmon';
+
+INSERT INTO vets VALUES (DEFAULT, 'William Tatcher', 45, '04-23-2000');
+INSERT INTO vets VALUES (DEFAULT, 'Maisy Smith', 26, 'Jan-17-2019');
+INSERT INTO vets VALUES (DEFAULT, 'Stephanie Mendez', 64, 'May-04-1981');
+INSERT INTO vets VALUES (DEFAULT, 'Jack Harkness', 38, 'Jun-08-2008');
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'William Tatcher'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'), 
+  (SELECT id FROM species WHERE species.name = 'Digimon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Jack Harkness'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
```

Comment:
- [ ] @Vgonma kindly ensure you enter the correct values the specialisation here is digimon
![image](https://user-images.githubusercontent.com/61408860/220998191-2fdb712d-816c-48ca-9d50-66ea40c4f568.png)